testapi.pm

```diff
@@ -2232,7 +2232,8 @@ sub upload_logs {
     bmwqemu::log_call(file => $file, failok => $failok, timeout => $timeout, %args);
     my $basename = basename($file);
     my $upname   = $args{log_name} || ($autotest::current_test->{name} . '-' . $basename);
-    my $cmd      = "curl --form upload=\@$file --form upname=$upname ";
+    # Redirect the output so the progress meter is displayed 
+    my $cmd = "curl -o /dev/$serialdev --form upload=\@$file --form upname=$upname ";
```

Comment:
this command is used *a lot*. And adding more to type can have a big accumulated effect. I would rather not add that by default but only when it's necessary. Maybe this is something you want to optionally enable with a variable?

Reply:
I chose a variable called `UPLOAD_PROGRESS` - is there a place for documenting it?

Reply:
the place that every test writer should actually know :) https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc